kafka-producers/values-run-different.yaml

```diff
@@ -0,0 +1,15 @@
+producers:
+  "auxtel":
```

Comment:
any reason for the quotes?  I don't think it should matter, it just looks a bit weird.

Reply:
Hmm. Don't know why that didn't work before, but I went back and removed the quotes and now it works. I'll update that.